CMakeLists.txt

```diff
@@ -199,6 +225,29 @@ if(PDC_SUPPRESS_UNUSED_BUT_SET)
   set(SUPPRESSED_LIST "-Wno-unused-but-set-variable" ${SUPPRESSED_LIST})
 endif()
 
+# Disable warnings for -Wsign-compare
+option(PDC_SUPPRESS_SIGN_COMPARE "Disable sign-compare warnings" ON)
```

Comment:
Do we need all of these options as PDC_ ?

Reply:
we are not trying to iterate every single warning options here, but the ones that are controlled by switches are the ones either too verbose or too trivial.

Reply:
But having them names PDC_sth is strange as it is not a PDC-feature.

Reply:
addressed